Dockerfile

```diff
@@ -0,0 +1,25 @@
+# STEP 1 build executable binary
```

Comment:
Why do you need a separate builder? Why do you download filtron and build it after instead of running `go get` without `-d`?